integrations/virustotal.py

```diff
@@ -190,109 +174,153 @@ def request_virustotal_info(alert: any, apikey: str):
-        else:
-          alert_output["virustotal"]["error"]         = response.status_code
-          alert_output["virustotal"]["description"]   = "Error: API request fail"
-          send_msg(json.dumps(alert_output))
-          raise Exception("# Error: VirusTotal credentials, required privileges error")
+        alert_output["virustotal"]["description"]   = "Error: API request fail"
+        error = 'Error: VirusTotal credentials, required privileges error'
+
+    send_msg(json.dumps(alert_output))
+
+    logger.error(error)
+    raise Exception(error)
 
 def send_msg(msg: any, agent:any = None) -> None:
+    logger.info("Sending message")
+
     if not agent or agent["id"] == "000":
         string      = '1:virustotal:{0}'.format(json.dumps(msg))
     else:
         location    = '[{0}] ({1}) {2}'.format(agent["id"], agent["name"], agent["ip"] if "ip" in agent else "any")
         location    = location.replace("|", "||").replace(":", "|:")
         string      = '1:{0}->virustotal:{1}'.format(location, json.dumps(msg))
 
-    debug("# Request result from VT server: %s" % string)
+    logger.debug("Event: %s", string)
+
     try:
         sock = socket(AF_UNIX, SOCK_DGRAM)
         sock.connect(SOCKET_ADDR)
         sock.send(string.encode())
         sock.close()
     except FileNotFoundError:
-        debug("# Error: Unable to open socket connection at %s" % SOCKET_ADDR)
+        logger.error("Error: Unable to open socket connection at %s", SOCKET_ADDR)
         sys.exit(ERR_SOCKET_OPERATION)
 
 def get_json_alert(file_location: str) -> any:
-    """
-        Read JSON alert object from file
-
-        Parameters
-        ----------
-        file_location : str
-            Path to the JSON file location.
-
-        Returns
-        -------
-        {}: any
-            The JSON object read it.
-
-        Raises
-        ------
-        FileNotFoundError
-            If no JSON file is found.
-        JSONDecodeError
-            If no valid JSON file are used
+    """Read JSON alert object from file
+
+    Parameters
+    ----------
+    file_location : str
+        Path to the JSON file location.
+
+    Returns
+    -------
+    any
+        JSON encoded alert.
+
+    Raises
+    ------
+    FileNotFoundError
+        If no JSON file is found.
+    JSONDecodeError
+        If no valid JSON file is used.
     """
     try:
-        with open(file_location) as alert_file:
+        with open(file_location, encoding='utf-8') as alert_file:
             return json.load(alert_file)
     except FileNotFoundError:
-        debug("# JSON file for alert %s doesn't exist" % file_location)
+        logger.error("JSON file for alert %s doesn't exist", file_location)
         sys.exit(ERR_FILE_NOT_FOUND)
     except json.decoder.JSONDecodeError as e:
-        debug("Failed getting JSON alert. Error: %s" % e)
+        logger.error("Failed getting JSON alert. Error: %s", e)
         sys.exit(ERR_INVALID_JSON)
 
+def print_help_msg():
```

Comment:
You included a comment for this function in other integrations, I would also include it here.